### PR TITLE
workflow cleanup in progress

### PR DESCRIPTION
**Pull request by birozuru**
Workflow cleanup in progress. This should fix on-going issues and provision stable pipeline for the future moving forward.

### CODE REVIEW

**Review by BatiGencho — DISMISSED** [review]

**Comment by arminsabouri** [comment]
Note: lets squash merge this PR

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -63,49 +65,51 @@ jobs:
...
-          --bitcoind-user ${{env.BITCOIND_USER}} \
-          --bitcoind-pass ${{ env.BITCOIND_PASS }} \
-          --fall-back-fee-rate-sat-per-vbyte 5 &
-          pid=$! 
-          sleep 150
-          echo "Killing BTC Server"
-          kill -9 $pid
+  # This performs the same test as the e2e job, but only on btc_server
+  # test:
+  #   name: test / ethereum
```
Comment (scottmillner) [inline .github/workflows/integration.yml]: the name of this test always confuses me

it only spins up btc-server but doesn't actually run any tests, right?

Comment (arminsabouri) [reply .github/workflows/integration.yml]: Yeah good point. We can probably remove this commented out workflow

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -63,49 +65,51 @@ jobs:
...
+  #         --bitcoind-user ${{env.BITCOIND_USER}} \
+  #         --bitcoind-pass ${{ env.BITCOIND_PASS }} \
+  #         --fall-back-fee-rate-sat-per-vbyte 5 &
+  #         pid=$! 
+  #         sleep 150
+  #         echo "Killing BTC Server"
+  #         kill -9 $pid
 
   sync:
     name: sync / 100k blocks
```
Comment (scottmillner) [inline .github/workflows/integration.yml:82]: same here as above, im confused, what am i missing?

both the `test / ethereum` and `sync / 100k blocks` jobs just run this command which starts up btc-server and nothing else: `/home/runner/.cargo/bin/cargo run --bin btc-server`

File: .github/workflows/integration.yml (hunk 1 above)
Comment (birozuru) [reply .github/workflows/integration.yml]: yeah that's why I commented it out cause it only spins up the btc-server

**Pull request merged**